scripts/package/postinstall.sh

```diff
@@ -50,11 +50,20 @@ If you have any other questions please contact us at support@observiq.com
 EOF
 }
 
+manage_sysv_service() {
+  chmod 755 /etc/init.d/observiq-otel-collector
+  chmod 644 /etc/sysconfig/observiq-otel-collector
+  echo "configured sysv service"
+}
+
 init_type() {
-  systemd_test="$(systemctl 2>/dev/null || : 2>&1)"
-  if command printf "$systemd_test" | grep -q '\-.mount'; then
+  # Determine if we need service or systemctl for prereqs
+  if command -v systemctl > /dev/null 2>&1; then
```

Comment:
This seems safe but I want to be sure. I cannot recall why we would parse the output of systemctl. What do you think?

Reply:
We used this method elsewhere. If it is safe elsewhere, I would think it is safe here as well.